Source/Core/HermiteSpline.js

```diff
@@ -346,7 +362,7 @@ HermiteSpline.createC1 = function (options) {
  * @param {Object} options Object with the following properties:
  * @param {Number[]} options.times The array of control point times.
  * @param {Cartesian3[]} options.points The array of control points.
- * @returns {HermiteSpline|LinearSpline} A hermite spline or a linear spline if less than 3 control points were given.
+ * @returns {HermiteSpline|LinearSpline} A hermite spline, or a linear spline if less than 3 control points were given.
```

Comment:
Sorry, this was leftover from my attempt to make these functions adapt to quaternions. The problem with them is that the functions used to generate tangents rely on `TridiagonalSystemSolver`, which is written to handle `Cartesian3`s. It may be possible to reuse this to generate cubic quaternion splines but we would need to check the math and write additional unit tests, and I personally think it's out of scope for this PR.

Reply:
So for now, `HermiteSpline.create___` functions should only expect to handle `Cartesian3s`. I can write an issue to expand this in the future.